indicators/NO_GLAC_001/R/NO_GLAC_001.qmd

```diff
@@ -0,0 +1,1731 @@
+
+<!--# Text here -->
+
+I know of no concrete plans to update the glacier extent map, but with
+the advent of remote sensing, this is indeed possible to to, and quite
+easy as well probably.
+
+#### 2.3.1 Instructions for citing, using and accessing data
+
+<!--# Is the data openly available? If not, how can one access it? What are the key references to the datasets?   -->
+
+The glaciel extent maps are distributed under CC-BY 4.0. We have made a
+simplified (less columns) version of the dataset available on github in
+order to make the workflow more reproducible.
+
+## 3. Indicator properties
+
+### 3.1 Ecosystem Condition Typology Class (ECT)
+
+Area of glaciers is categorised as describing a *C1 - Landscape and
+seascape characteristic*. This puts emphasis in the role of glaciers as
+a unique habitat type and a structural element in alpine ecosystems. The
+indicator has previously been attributed to *A1 - Physical state
+characteristics* due to the importance of melt water.
+
+### 3.2 Ecosystem condition characteristic
+
+### 3.3 Other standards
+
+In the Norwegian standard [@nybø2017] I place the indicator under the
+category *Landskapsøkologiske mønstre*.
+
+### 3.4 Collinearities with other indicators
+
+The indicator is in many ways unique, but will likely be collinear with
+other indicators that are similarly affected by climate warming.
+
+### 3.5 Impact factors
+
+Glaciers are mainly affected by climate change, especially climate
+warming [@zemp2025].
+
+## 4. Reference condition and levels
+
+### 4.1 Reference condition
+
+The reference condition of mountain ecosystems are defined, in general
+terms, as one with minimal human impact. For the climate, the definition
+is more precisely defined as the time period 1961-1990, which is the
+last 30 year normal period before climate change became really
+prominent.
+
+### 4.2 Reference levels
+
+Reference levels are defined for each EA uniquely based on the glacier
+extent in 1947-1985.
+
+#### 4.2.1 Spatial resolution and validity
+
+The reference levels are defined for each EA, and can therefore only be
+used to normalise variables at that scale. However, reference levels can
+easily be recalculated for any spatial scale, as I have also done here.
+When testing out alternative aggregation pathways I had to create new
+reference levels for larger regions.
+
+## 5. Uncertainties
+
+There is some uncertainty in the mapping method,and this is said to be
+3%. There is also some uncertainty, or at least some arbitrariness, in
+how X~60~ is defined as 10% of X~100~.
+
+## 6. References
+
+::: {#refs}
+:::
+
+## 7. Datasets
+
+### 7.1 Dataset A - Glacier cover 1947-1985
+
+This data is based on a combination of landsat imagery and digitized
+topographic maps [@winsvold2014]. We use this map as our reference
+condition and assume this glacier cover represents the climatic period
+1961-1990.
+
+The dataset is manually downloaded.
+
+URL: https://nve.brage.unit.no/nve-xmlui/handle/11250/2831053
```

Comment:
This url is not working in the html. Same for dataset B

Reply:
I have fixed this and for dataset B

Reply:
I will making the link active